bin/control.sh

```diff
@@ -42,6 +42,9 @@ export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 export ALERT_ENGINE_MAX_HEAP=${ALERT_ENGINE_MAX_HEAP:-"2G"}
 export COLLECTION_ROLLER_MAX_HEAP=${COLLECTION_ROLLER_MAX_HEAP:-"2G"}
 export LOG_COLLECTOR_MAX_HEAP=${LOG_COLLECTOR_MAX_HEAP:-"8G"}
+export KAFKA_PROPS=${KAFKA_PROPS:-"kafka.properties"}
+export KAFKA_OPTS="-Djava.security.auth.login.config=${CONF_DIR}/jaas.conf"
+export KAFKA_TOPIC=${KAFKA_TOPIC:-"pulse-kafka-topic"}
```

Comment:
I think we should leave this empty. Having a default makes it easier to not add the property, it fails, and the user wonders where this topic name came from

Reply:
Is there somewhere in the service.sdl where the KAFKA_TOPIC is set?

Reply:
> Is there somewhere in the service.sdl where the KAFKA_TOPIC is set?
Yea. It is set in service.sdl